Documentation/ColumnsConfig/Type/Slug/Examples.rst

```diff
@@ -1,27 +1,40 @@
-.. include:: /Includes.rst.txt
-.. _columns-slug-examples:
+..  include:: /Includes.rst.txt
+..  _columns-slug-examples:
 
 ========
 Examples
 ========
 
-.. _tca_example_slug_1:
+..  versionadded:: 12.0
+    When using the `slug` type, TYPO3 takes care of generating the necessary
+    TCA configuration. Developers only need to define the TCA column and add
```

Comment:
Does it? At least the change you linked here only talks about the creation of the database field, not the TCA configuration?